src/Sylius/Bundle/CoreBundle/Resources/config/services.xml

```diff
@@ -408,11 +408,15 @@
             <tag name="validator.constraint_validator" alias="registered_user_validator" />
         </service>
 
-        <service id="sylius.custom_factory.cart_item" class="Sylius\Component\Core\Factory\CartItemFactory" decorates="sylius.factory.cart_item">
-            <argument type="service" id="sylius.custom_factory.cart_item.inner" />
+        <service id="sylius_core.factory.cart_item" class="Sylius\Component\Core\Factory\CartItemFactory" decorates="sylius.factory.cart_item" public="false">
```

Comment:
If we have `sylius` as first key word for the whole system, I think we should stay with it, and come up with some better naming after first dot